<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(65, 65, 65)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(180, 180, 180, 255))</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/logo_720.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>801</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>213</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="Username">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Username...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Password">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="error">
        <property name="minimumSize">
         <size>
          <width>187</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(255, 255, 255, 0);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_login">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);

}
QPushButton:hover {
color:black;
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

}</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="forgetpassword">
        <property name="styleSheet">
         <string notr="true"> background-color:rgba(255, 255, 255, 0);
font: 10pt &quot;Times New Roman&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Forgot Password?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>Password</zorder>
     <zorder>Username</zorder>
     <zorder>error</zorder>
     <zorder>forgetpassword</zorder>
     <zorder></zorder>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>78</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>78</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>78</width>
       <height>31</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0);
color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
